tests/lava/lib/optimization/solvers/generic/monitoring_processes/solution_readout/test_solution_readout.py

```diff
@@ -22,12 +22,15 @@ class TestSolutionReadout(unittest.TestCase):
     def setUp(self) -> None:
         # Create processes.
         spiker = Spiker(shape=(4,), period=3, payload=7)
-        integrator = Spiker(shape=(1,), period=7, payload=-4)

```

Comment:
here the comment is totally necessary! Thanks for it. An alternative would be to have a couple of intermediate variables that show how you derived each component.